xmodule/js/src/video/036_video_social_sharing.js

```diff
@@ -0,0 +1,111 @@
+(function(define) {
+    'use strict';
+    // VideoSocialSharingHandler module.
+    define(
+        'video/036_video_social_sharing.js', ['underscore', 'gettext'],
+        function(_, gettext) {
+            var VideoSocialSharingHandler, SocialSharingSite, SimpleSocialSharingSite, facebook, linkedin, twitter;
+            /**
+     * Video Social Sharing control module.
+     * @exports video/036_video_social_sharing.js
+     * @constructor
+     * @param {jquery Element} element
+     * @param {Object} options
+     */
+            VideoSocialSharingHandler = function(element, options) {
+                if (!(this instanceof VideoSocialSharingHandler)) {
+                    return new VideoSocialSharingHandler(element, options);
+                }
+
+                _.bindAll(this, 'clickHandler');
+
+                this.container = element;
+
+                if (this.container.find('.wrapper-downloads .wrapper-social-share')) {
+                    this.initialize();
+                }
+
+                return false;
+            };
+
+            VideoSocialSharingHandler.prototype = {
```

Comment:
remove link generation. just read from the href

Reply:
Fixed in https://github.com/openedx/edx-platform/pull/32187